packages/core/src/components/workflow-steps/workflow-steps.tsx

```diff
@@ -65,25 +65,49 @@ export class WorkflowSteps {
     });
   }
 
-  componentDidRender() {
-    const steps = this.getSteps();
-
+  styling(extraSteps: HTMLIxWorkflowStepElement[] = []) {
+    let steps = [...this.getSteps(), ...extraSteps];
     steps.forEach((element, index) => {
-      element.setAttribute(
-        'vertical',
-        this.vertical === true ? 'true' : 'false'
-      );
-      element.setAttribute(
-        'clickable',
-        this.clickable === true ? 'true' : 'false'
-      );
+      element.setAttribute('vertical', this.vertical ? 'true' : 'false');
+      element.setAttribute('clickable', this.clickable ? 'true' : 'false');
       element.setAttribute(
         'selected',
         this.selectedIndex === index ? 'true' : 'false'
       );
       if (index === 0) element.setAttribute('position', 'first');
       if (index === steps.length - 1) element.setAttribute('position', 'last');
+      if (index > 0 && index < steps.length - 1)
+        element.setAttribute('position', 'undefined');
+    });
+  }
+
+  private observer: MutationObserver;
+
+  componentDidLoad() {
+    const steps: HTMLIxWorkflowStepElement[] = [];
+    const mList = this.hostElement.querySelector('.steps');
+    const options = {
```

Comment:
avoid any here use the correct typing `MutationObserverInit` or remove to options and set it inline inside the observe function